commit d235661b6c30b4002f81c63cebe744ba3edbb453
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Apr 4 16:39:44 2018 +0100

    lan78xx: Don't reset the interface on open
    
    With Alexander Graf's patch ("lan78xx: Connect phy early") applied,
    the call to lan78xx_reset within lan78xx_open prevents the phy
    interrupt from being generated (even though the link is up).
    
    Avoid this issue by removing the lan78xx_reset call.
    
    See: https://github.com/raspberrypi/linux/issues/2437
         https://github.com/raspberrypi/linux/issues/2442
         https://github.com/raspberrypi/linux/issues/2457

diff --git a/drivers/net/usb/lan78xx.c b/drivers/net/usb/lan78xx.c
index 7cbaefa2635b..3f188af4b236 100644
--- a/drivers/net/usb/lan78xx.c
+++ b/drivers/net/usb/lan78xx.c
@@ -2553,10 +2553,6 @@ static int lan78xx_open(struct net_device *net)
 	if (ret < 0)
 		goto out;
 
-	ret = lan78xx_reset(dev);
-	if (ret < 0)
-		goto done;
-
 	phy_start(net->phydev);
 
 	netif_dbg(dev, ifup, dev->net, "phy initialised successfully");
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1232_d235661b6c30b4002f81c63cebe744ba3edbb453.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1232_d235661b6c30b4002f81c63cebe744ba3edbb453.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
